site/content/user-guide/amqp-adapter.md

```diff
@@ -38,20 +38,20 @@ There is a subtle difference between the *device identifier* (*device-id*) and t
 {{% /note %}}
 
 ### SASL EXTERNAL Authentication
-When a device uses a client certificate for authentication using SASL EXTERNAL, the AMQP adapter internally performs the following steps:
+When a device uses a client certificate for authentication, the TLS handshake is initiated as part of the SASL authentication process. If no trust anchor is configured for the AMQP adapter, the TLS handshake will succeed only if the certificate has not yet expired. The check for the certificate's signature does not happen during the TLS handshake and is deferred to later stages. If the TLS handshake succeeds, the adapter does the following to complete the SASL EXTERNAL authentication process:
```

Comment:
In fact, the TLS handshake takes place *before* the SASL handshake can even start. That is because the SASL handshake requires an established  TCP connection ...

the check of the signature is performed as part of the SASL handshake, isn't it? why not make this explicit?